app/Http/Serializers/IpmaWarningSerializer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace VOSTPT\Http\Serializers;
+
+use Tobscure\JsonApi\AbstractSerializer;
+
+class IpmaWarningSerializer extends AbstractSerializer
+{
+    /**
+     * {@inheritDoc}
+     */
+    protected $type = 'ipma_warnings';
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getAttributes($model, array $fields = null): array
+    {
+        return [
+            'uuid'                => $model->uuid,
+            'text'                => $model->text,
+            'awareness_type_name' => $model->awareness_type_name,
+            'awareness_level_id'  => $model->awareness_level_id,
+            'zone_name'           => $model->zone_name,
```

Comment:
Replace zone/area with the proper `County`

Reply:
District , its not possible to map to Country.

Reply:
The zone must be present , otherwise we will lose the ability to know if the warning is for the Continent / Madeira or Açores. Maybe zone its not the best name, any suggestion ?